storage/innobase/include/trx0trx.h

```diff
@@ -715,12 +715,31 @@ struct trx_t : ilist_node<>
   alignas(CPU_LEVEL1_DCACHE_LINESIZE) trx_lock_t lock;
 
 #ifdef WITH_WSREP
-  /** whether wsrep_on(mysql_thd) held at the start of transaction */
-  byte wsrep;
+  /** wsrep_on(mysql_thd) held at the start of transaction */
+  static constexpr byte TRX_WSREP_ON = byte{1};
+  /** wsrep_thd_is_BF(mysql_thd) held at start of transaction */
+  static constexpr byte TRX_WSREP_BF = byte{2};
+  /** BF thread is performing unique secondary index scan */
+  static constexpr byte TRX_WSREP_UK_SCAN = byte{4};
+  /** whether this transaction is running wsrep transaction with following sub-states:
+      (1) whether wsrep_on(mysql_thd) held at the start of transaction
+      (2) whether wsrep_thd_is_BF(mysql_thd) held at start of transaction
+      (4) whether BF thread is performing unique secondary index scan
```

Comment:
The `TRX_` prefix is redundant, because these are `static` members of `trx_t`.

The comment of `wsrep` could simply mention that it is a combination of the flags `WSREP_ON`, `WSREP_BF`, `WSREP_UK_SCAN`. The semantics of those flags is already documented in the per-flag comments.

Reply:
Done